slamd/templates/landing_page.html

```diff
@@ -9,6 +9,9 @@
 {% extends 'index.html' %}
 {% block content %}
 <main class="col-sm-12 col-md-12 col-lg-12 ml-sm-auto px-md-2 py-2 container">
+<!-- Landing page does not contain any forms. In order to make navbar work anyway, manually add CSRF token. -->
+    <input type="hidden" id="csrf_token" value="{{ csrf_token() }}"/>
+
```

Comment:
Does it? I specifically added the hidden field on the landing page and not in the navbar/index.html so that it would _not_ appear twice on every page. This should just add the "missing" token to the landing page, which does not have a token element to retrieve otherwise, because it does not contain any forms.

Reply:
As discussed in the weekly and on teams keeping the CSRF token here is the simplest solution. I added comments in the navbar form and the SessionController file to point out this special case.